IdentityCore/src/cache/token/MSIDCredentialCacheItem.m

```diff
@@ -115,6 +115,8 @@ - (NSUInteger)hash
     hash = hash * 31 + self.refreshOn.hash;
     hash = hash * 31 + self.cachedAt.hash;
     hash = hash * 31 + self.lastRecoveryAttempt.hash;
+    hash = hash * 31 + self.recoveryAttemptCount.hash;
```

Comment:
Let's not include it here, or it will make the jti be changed between attempts and who knows what issues it will cause

Reply:
So we remove both recovery attempt count and recovery attempt failed flag both from the hash function right

Reply:
I'd recommend that, yes, even lastRecoveryAttempt as well, since now we'll try those twice, and it might cause trouble

Reply:
OK will do

Reply:
Removed